Source/WebCore/css/counterStyles.css

```diff
@@ -285,14 +287,14 @@
 }
 
 @counter-style cjk-earthly-branch {
-    system: fixed;
+    system: alphabetic;
     symbols: "\5B50" "\4E11" "\5BC5" "\536F" "\8FB0" "\5DF3" "\5348" "\672A" "\7533" "\9149" "\620C" "\4EA5";
     /* 子 丑 寅 卯 辰 巳 午 未 申 酉 戌 亥 */
     suffix: "\3001";
 }
 
 @counter-style cjk-heavenly-stem {
-    system: fixed;
+    system: alphabetic;
```

Comment:
Same question.

Reply:
The specification says this system is `fixed`, however the related WPT tests expect alphabetic. Alphabetic does make sense, otherwise it would cover a range of only 11 values (the number of symbols) before falling back to decimal. The other browsers behave as `alphabetic` as well. I'm opening a spec issue related to that.

Reply:
https://github.com/w3c/csswg-drafts/issues/8596